acsys/dpm/__init__.py

```diff
@@ -86,7 +86,7 @@ def isStatusFor(self, *tags):
         warnings.warn(
             "deprecated in favor of the snake_case version, is_status_for",
             DeprecationWarning)
-        return self.is_status_for
+        return self.is_status_for()
```

Comment:
Is this correct? Shouldn't it pass `*tags`?

Reply:
Possibly, I don't think it matters since no user should ever call this.

Reply:
To say more, because this is in a "private" class `_ItemCommon`, this only defines the shape of the inherited classes, so the argument signature is what needs to be correct.
Notice that the function that is called doesn't use the `tags` argument.